components/browser/storage-sync/src/test/java/mozilla/components/browser/storage/sync/PlacesHistoryStorageTest.kt

```diff
@@ -485,12 +486,13 @@ class PlacesHistoryStorageTest {
                 return mock()
             }
 
-            override fun syncHistory(syncInfo: SyncAuthInfo) {
+            override fun syncHistory(syncInfo: SyncAuthInfo): SyncTelemetryPing? {
```

Comment:
You can tell I didn't add tests yet, because this won't compile. 😅

Reply:
Have you considered writing unit tests for your metrics? Glean offers a [testing API](https://github.com/mozilla-mobile/android-components/blob/master/components/service/glean/docs/metrics/testing-metrics.md), which would make your life easier for validating assumptions on the metrics you're landing

Reply:
With help from @travis79 and @mdboom, I got tests working using a mock server. Unfortunately, I couldn't use `testGetValue` because I'd need to change the interface to allow intercepting the ping before it was submitted. 😞